user_guide_src/source/models/model.rst

```diff
@@ -187,7 +187,7 @@ data type.
 
 **$createdField**
 
-Specifies which database field should use for keep data record create timestamp.
+Specifies which database field to use for data record create timestamp.
 Leave it empty to avoid update it (even useTimestamps is enabled)
```

Comment:
```suggestion
Leave it empty to avoid updating it (even if ``$useTimestamps`` is enabled)
```

Reply:
Yes, very good, see my previous comment above.  If you would like me to continue with these "nitty-gritties" I'd be delighted to do so as I peruse the docs and notice these anomalies.  I'll try to get better at the commits, and try to be better about the descriptions. Please give me feedback as required.
Again, aside from these little hiccups, I'm very impressed by the docs!

- [ ]

Reply:
Please do so. Your help is appreciated.